docs/application/native/guides/security/device-certificate-manager.md

```diff
@@ -0,0 +1,138 @@
+# Device Certificate Manager
+
+The Device Certificate Manager (DCM) provides cryptography services (digital certificates and keys) for authentication and secure communication with another system. Device certificate and key was embedded in device storage.
+
+The main features of the Device Certificate Manager API include:
+
+- Getting key type/length
+
+- Getting certificate chain
+
+- Creating digital signatures
+
+## Prerequisites
+
+To enable your application to use the Device Certificate Manager functionality:
+
+1. To use the Device Certificate Manager API (in [mobile](../../api/mobile/latest/group__CAPI__DEVICE__CERTIFICATE__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__DEVICE__CERTIFICATE__MANAGER__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
To use the Device Certificate Manager API (in [mobile](../../api/mobile/latest/group__CAPI__DEVICE__CERTIFICATE__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__DEVICE__CERTIFICATE__MANAGER__MODULE.html) applications), the application must request permission by adding the following privilege to the `tizen-manifest.xml` file: